src/js/TaskService.js

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+
+function Crud({ priority, text, id, completed}) {
+  this.priority = priority;
+  this.text = text;
+  this.id = id;
+  this.completed = completed;
+}
+
+export default class TaskService {
+  getAll (cruds) {
```

Comment:
pogledaj na onom projektu sto sam ti poslao kako je napisan servis. Mislim da ne bi trebalo ovde da brises iz lokalne kolekcije todos, vec da to radis u komponenti.